File: python/mix/C01_fib_type1_a.py
def fib(n: int) -> int:
    """Return the n-th Fibonacci number (0-indexed)."""
    a, b = 0, 1
    for _ in range(n):
        a, b = b, a + b
    return a

if __name__ == "__main__":
    print(fib(10))

File: javascript/mix/C01FibType1A.js
function fib(n) {
    let a = 0, b = 1;
    for (let i = 0; i < n; i++) {
        const temp = a;
        a = b;
        b = temp + b;
    }
    return a;
}

console.log(fib(10));

module.exports = { fib };
